sarracen/render.py

```diff
@@ -215,6 +216,8 @@ def render(data: 'SarracenDataFrame',  # noqa: F821
     corotation: list, optional
         Moves particles to the co-rotating frame of two location. corotation
         contains two lists which correspond to the two x, y, z coordinates
+    plot_apr: bool
+        Plots the APR regions on top of the rendered quantity if True
```

Comment:
This should be an optional parameter in the docstring (i.e., `plot_apr: bool, optional`) and specify in the text that the default is False.